src/ui/views/ViewSubscriptionNeeded.qml

```diff
@@ -170,21 +170,30 @@ VPNFlickable {
                 }
             }
 
+            VPNLinkButton {
+                id: restorePurchase
+                visible: Qt.platform.os == "ios"
+
+                // Already a subscriber?
+                labelText: qsTrId("vpn.updates.alreadyASubscriber")
```

Comment:
We need to add this to strings.yaml and call it with VPNl18n

Reply:
Maybe this string already exists.

Reply:
@bakulf Where can I check if the string already exists?

Reply:
4.) Check the[ l10n repo](https://github.com/mozilla-l10n/mozilla-vpn-client-l10n)
Here is the Pontoon project: https://pontoon.mozilla.org/projects/mozilla-vpn-client/